bench/tiered/push_pull.c

```diff
@@ -204,6 +210,69 @@ fill_random_data(void)
     data_str[str_len - 1] = '\0';
 }
 
+/*
+ * remove_local_cached_files --
+ *     Remove local cached files and cached folders.
+ */
+static void
+remove_local_cached_files(const char *home)
+{
+    struct dirent *dir_entry;
+    DIR *dir;
+
+    char *tablename;
+    char delete_file[512], file_prefix[1024], rm_cmd[512];
+    int highest, index, nmatches, objnum, status, success_status;
+
+    nmatches = highest = objnum = success_status = 0;
```

Comment:
Alphabetize on this line.

Reply:
Fixed in 4eb0e1bc7e8a9c3651cc9be94d4f7e11f2520ac5